ssh-key/src/signature.rs

```diff
@@ -517,6 +519,15 @@ impl_signature_for_curve!(p256, "p256", NistP256, 32);
 impl_signature_for_curve!(p384, "p384", NistP384, 48);
 impl_signature_for_curve!(p521, "p521", NistP521, 66);
 
+#[cfg(any(feature = "p256", feature = "p384", feature = "p521"))]
+fn build_field_bytes<B: FromIterator<u8> + Copy>(m: Mpint) -> Option<B> {
```

Comment:
Perhaps this should be called something like:

```suggestion
fn zero_pad_field_bytes<B: FromIterator<u8> + Copy>(m: Mpint) -> Option<B> {
```

Reply:
Sure, done in baa16c27ecf03a44811fbbde7ce74e49307c9903 and added a docstring in 0b0421846839ce22f24b8e9ef8e849774fb1ec3f